packages/jaeger-ui/src/model/ddg/Graph.test.js

```diff
@@ -63,37 +63,68 @@ describe('Graph', () => {
 
   describe('getVertexKey', () => {
     const testFocalElem = simpleModel.paths[0].members[2];
-    const expectedKeyEntry = pathElem => `${pathElem.operation.service.name}----${pathElem.operation.name}`;
-    const expectedFocalElemKey = expectedKeyEntry(testFocalElem);
     // Because getVertexKey is completely context-unaware until late-alpha, an empty ddg is sufficient to test
     // this method.
-    const emptyGraph = new Graph({ ddgModel: { visIdxToPathElem: [] } });
+    const ddgModel = { visIdxToPathElem: [] };
 
-    it('creates key for focal pathElem', () => {
-      expect(emptyGraph.getVertexKey(testFocalElem)).toBe(expectedFocalElemKey);
-    });
+    describe('showOp is true', () => {
```

Comment:
this is DRYed out in [next PR](https://github.com/jaegertracing/jaeger-ui/pull/433)